### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by DragonKyro**
Python unittests for VR connection, DJI Creation, Mavros Creation, DJI control tasks, and Mavros control tasks.

### CODE REVIEW

File: src/test.py [hunk 1]
```diff
@@ -0,0 +1,603 @@
+import unittest
+import roslibpy
```
Comment (erwang01) [inline src/test.py:2]: Would be nice to add a timeout: https://pypi.org/project/timeout-decorator/

File: src/operator.py [hunk 2]
```diff
@@ -241,6 +248,14 @@ def register_drone(request, response):
     print("Calling register_drone service...")
     drone_name = request["drone_name"]
     drone_type = request["drone_type"]
+
+    if drone_name in drone_names:
+        response["success"] = False
+        response["message"] = "A drone with this name already exists."
+        response["id"] = 0
+        print("A drone with this name already exists")
```
Comment (erwang01) [inline src/operator.py:256]:
```suggestion
        print("A drone with the name", drone_name, "already exists")
```

File: src/operator.py [hunk 3]
```diff
@@ -241,6 +248,14 @@ def register_drone(request, response):
...
+        return True
```
Comment (erwang01) [inline src/operator.py:257]: Does return False not work? I don't remember if its True only on service success or just always.

File: src/operator.py [hunk 4]
```diff
@@ -335,6 +350,14 @@ def register_sensor(request, response):
     sensor_name = request["sensor_name"]
     sensor_type = request["sensor_type"]
     parent_drone_name = request["parent_drone_name"]
+
+    if sensor_name in sensor_names:
+        response["success"] = False
+        response["message"] = "A sensor with this name already exists."
+        response["id"] = 0
+        print("A sensor with this name already exists")
```
Comment (erwang01) [inline src/operator.py:358]:
```suggestion
        print("A sensor with the name", sensor_name, "already exists")
```

**Review by erwang01 — COMMENTED** [review]
LGTM a few changes, let me know what you think.

**Comment by erwang01** [comment]
Also may be worth investigating using test suites to organize test cases along with setup and teardown methods.

File: src/test.py (hunk 1 above)
Comment (erwang01) [reply src/test.py:2]: Added. This does cause some issues with the server tho... Although I think our server should be robust to this so its okay for now.

File: src/operator.py (hunk 3 above)
Comment (DragonKyro) [reply src/operator.py:257]: I tried returning False and the service just hung. So I think it has to return True.

**Review by DragonKyro — COMMENTED** [review]
Test cases reviewed.  Created 'make working' to run currently working services.  Ready to be merged.

**Pull request closed without merging**